addon/mixins/table.js

```diff
@@ -53,23 +53,34 @@ export default Mixin.create({
   /**
    * Pre-computed indices
    * @param {Boolean} haveCategories - whether the table header has categories or not
+   * @returns {String} jQuery selector string for getting the header column row
+   */
+  headerColumnsSelector (haveCategories) {
+    return `${haveCategories ? '.frost-table-header-columns' : ''}`
```

Comment:
Why isn't this code just `return haveCategories ? '.frost-table-header-columns' : ''`?